siliconcompiler/tools/surelog/surelog.py

```diff
@@ -109,51 +109,109 @@ def runtime_options(chip):
 
     cmdlist = []
 
-    # Deduplicated source files
-    # Library directories.
+    #####################
+    # Library directories
+    #####################
+
     ydir_files = chip.find_files('option', 'ydir')
+
+    for item in chip.getkeys('library'):
+        ydir_files.extend(chip.find_files('library', item, 'option', 'ydir'))
```

Comment:
Seems OK, although I wonder if how exactly soft libraries are incorporated requires more thought. Seems unrelated to main focus of PR.